tutorials/space_shooter/README.md

```diff
@@ -1,3 +1,55 @@
-# Flame tutorial
+# Flame Tutorial
+ABOUT:
+In this Flame tutorial we will provide you with the foundational steps needed to build several parts of the engine of the space shooter game.
+
+This README contains useful videos and articles from the community for this project. The links for these can be found under "Highlighted Community Tutorials."
+
+*Installation information*
+
+<ul>
+    <li>Need Flutter SDK installed for Android platform</li>
+    <li>Feel Free to use an IDE of your choice.</li>
+</ul>
+
+# Part 1
+Link for Part1 video tutorial: https://youtu.be/VEfZgTGolPY 
```

Comment:
I don't think we want links to non-official videos directly in the official tutorial, you could add it as a link together with the other links though.

Reply:
Sounds good!